src/services/index.js

```diff
@@ -5,7 +5,10 @@ import RoleService from './roleService';
 import SupplierService from './supplierService';
 import FacilityService from './facilityService';
 import RoomService from './roomService';
+import RequestService from './requestService';
 
 export {
-  UserService, BookingService, CompanyService, SupplierService, RoleService, FacilityService, RoomService
+  UserService, BookingService, CompanyService,
```

Comment:
We can have each service on its own line to improve readability

Reply:
I will update this right away